src/elexmodel/models/GaussianElectionModel.py

```diff
@@ -83,6 +90,13 @@ def get_unit_prediction_intervals(self, reporting_units, nonreporting_units, alp
             lower.round(decimals=0), upper.round(decimals=0), prediction_intervals.conformalization
         )
 
+    def get_conformalization_data_unit(self):
+        return self.gaussian_bounds_unit, self.conformalization_data_unit
+
+    # Note that precinct level % conformalization data
```

Comment:
should be unit level not precinct. Also please add as docstring

Reply:
Done and clarified